backend/Watchdog.Core/Watchdog.Core.API/Controllers/MembersController.cs

```diff
@@ -17,34 +18,72 @@ public MembersController(IMemberService memberService)
             _memberService = memberService;
         }
 
-        [HttpGet]
-        public async Task<ActionResult<ICollection<MemberDto>>> GetAllAsync()
+        [HttpGet("organization/{organizationId}")]
+        public async Task<ActionResult<ICollection<MemberDto>>> GetAllMembers(int organizationId)
         {
-            var members = await _memberService.GetAllMembersAsync();
+            var members = await _memberService.GetMembersByOrganizationIdAsync(organizationId);
             return Ok(members);
         }
 
-        [HttpGet("{id:int}")]
-        public async Task<ActionResult<MemberDto>> GetByIdAsync(int id)
+        [HttpGet("{id}")]
+        public async Task<ActionResult<MemberDto>> GetMemberById(int id)
         {
             var member = await _memberService.GetMemberByIdAsync(id);
             return Ok(member);
         }
 
-        [HttpPost]
-        public async Task<ActionResult<MemberDto>> CreateMemberAsync(MemberDto memberDto)
+        [HttpPut]
+        public async Task<ActionResult<MemberDto>> Update(UpdateMemberDto dto)
         {
-            var member = await _memberService.CreateMemberAsync(memberDto);
+            var member = await _memberService.UpdateAsync(dto);
             return Ok(member);
         }
 
-        [HttpGet("organization/{organizationId}")]
-        public async Task<ActionResult<ICollection<MemberDto>>> GetAllMembersByTeam(int organizationId)
+        [HttpPost]
+        public async Task<ActionResult> AddMember(NewMemberDto newMemberDto)
         {
-            var members = await _memberService.GetMembersByOrganizationIdAsync(organizationId);
+            MemberDto member;
+            try
+            {
+                member = await _memberService.AddMemberAsync(newMemberDto);
+            }
+            catch (ArgumentException ex)
+            {
+                return Conflict(ex.Message);
+            }
+            var response = await _memberService.InviteMemberAsync(member);
+            return Ok(new { Member = member, SendMailResponse = response.StatusCode });
+        }
+
+
+        [HttpDelete("{id}")]
+        public async Task<ActionResult> DeleteMember(int id)
+        {
+            await _memberService.DeleteMemberAsync(id);
+            return NoContent();
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<ICollection<MemberDto>>> GetAllAsync()
+        {
+            var members = await _memberService.GetAllMembersAsync();
             return Ok(members);
         }
 
+        [HttpPost("reinvite")]
+        public async Task<ActionResult<int>> Reinvite(InviteDto dto)
+        {
+            var response = await _memberService.InviteMemberAsync(dto.Id);
+            return Ok(response.StatusCode);
+        }
+
+        [HttpPost("acceptinvite")]
```

Comment:
Please use the same type of route naming as we have in other controllers

Reply:
Done